pkgs/clean-pkg/src/genie/libs/clean/stages/stages.py

```diff
@@ -1195,19 +1197,22 @@ def apply_configuration(section, steps, device, configuration=None,
                         .format(device.name))
 
     # Copy running-config to startup-config
-    with steps.start("Copy running-config to startup-config on device {}".\
-                     format(device.name)) as step:
-        try:
-            device.api.execute_copy_run_to_start(command_timeout=config_timeout,
-                                                 max_time=max_time,
-                                                 check_interval=check_interval,
-                                                 copy_vdc_all=copy_vdc_all)
-        except Exception as e:
-            step.failed("Failed to copy running-config to startup-config on "
-                           "{}\n{}".format(device.name, str(e)))
-        else:
-            step.passed("Successfully copied running-config to startup-config "
-                        "on {}".format(device.name))
+    if skip_copy_run_start:
```

Comment:
if not skip_copy_run_start:
            with steps.start("Copy running-config to startup-config on device {}".\
....

No need of the if pass

Reply:
Thank you for the feedback.  Change completed.